Directory.Build.props

```
@@ -0,0 +1,9 @@
+﻿<Project>
+    <PropertyGroup>
```

Seeing as you're modernizing, I'd strongly recommend setting Artifacts output layout too:
```suggestion
    <PropertyGroup>
      <UseArtifactsOutput>true</UseArtifactsOutput>
```

This puts all the output (bin/obj/packages) into a top level `artifacts` folder. I always recommend it where possible these days (_some_ things break with it due to invalid assumptions, but I think this repo should be fine AFAICT)